mllib/src/main/scala/org/apache/spark/ml/classification/LinearSVC.scala

```diff
@@ -47,7 +47,7 @@ private[classification] trait LinearSVCParams extends ClassifierParams with HasR
 /**
  * :: Experimental ::
  *
- * Linear SVM Classifier (https://en.wikipedia.org/wiki/Support_vector_machine#Linear_SVM)
+ * [[https://en.wikipedia.org/wiki/Support_vector_machine#Linear_SVM  Linear SVM Classifier]]
```

Comment:
Would you mind changing it to

```
 * <a href="https://en.wikipedia.org/wiki/Support_vector_machine#Linear_SVM">
 * Linear SVM Classifier</a>
```

? It seems scala link annotation breaks Javadoc8 as below:

```
[error] .../spark/mllib/target/java/org/apache/spark/ml/classification/LinearSVC.java:5: error: unexpected text
[error]  * {@link https://en.wikipedia.org/wiki/Support_vector_machine#Linear_SVM  Linear SVM Classifier}
```

I tested this via `./build/sbt unidoc` with Java 8.

(Currently, it seems there are already few errors. Let me fix them with other warnings in near future).